rb/lib/selenium/webdriver/edge_chrome/bridge.rb

```diff
@@ -22,7 +22,46 @@
 module Selenium
   module WebDriver
     module EdgeChrome
-      class Bridge < WebDriver::Chrome::Bridge
+      class Bridge < WebDriver::Remote::Bridge
+
+        COMMANDS = {
+          get_network_conditions: [:get, 'session/:session_id/chromium/network_conditions'],
+          set_network_conditions: [:post, 'session/:session_id/chromium/network_conditions'],
+          send_command: [:post, 'session/:session_id/ms/cdp/execute'],
+          get_available_log_types: [:get, 'session/:session_id/se/log/types'],
+          get_log: [:post, 'session/:session_id/se/log']
+        }.freeze
+
+        def commands(command)
+          COMMANDS[command] || super
+        end
+
+        def network_conditions
+          execute :get_network_conditions
+        end
+
+        def send_command(command_params)
+          execute :send_command, {}, command_params
+        end
+
+        def network_conditions=(conditions)
+          execute :set_network_conditions, {}, {network_conditions: conditions}
+        end
+
+        def available_log_types
+          types = execute :get_available_log_types
+          Array(types).map(&:to_sym)
+        end
+
+        def log(type)
+          data = execute :get_log, {}, {type: type.to_s}
+
+          Array(data).map do |l|
+            LogEntry.new l.fetch('level', 'UNKNOWN'), l.fetch('timestamp'), l.fetch('message')
+          rescue KeyError
+            next
+          end
+        end
```

Comment:
I believe it might be easier to simply override 1 command:

```ruby
module Selenium
  module WebDriver
    module EdgeChrome
      class Bridge < WebDriver::Chrome::Bridge

        COMMANDS = WebDriver::Chrome::Bridge::COMMANDS.merge(
          send_command: [:post, 'session/:session_id/ms/cdp/execute']
        ).freeze

      end # Bridge
    end # EdgeChrome
  end # WebDriver
end # Selenium
```

Reply:
Okay. I did the overriding way before.
d9575cf has this change. The functional test also passed